tensorflow/python/kernel_tests/linalg/sparse/BUILD

```diff
@@ -1,6 +1,11 @@
 # Tests of TensorFlow sparse linear algebra kernels using the Python API.
 
-load("//tensorflow:tensorflow.default.bzl", "cuda_py_strict_test")
+load("//tensorflow:strict.default.bzl", "py_strict_library")
```

Comment:
Loaded symbol "py_strict_library" is unused. Please remove it.

Reply:
Sure, done!